tests/testthat/test-compactness.R

```diff
@@ -7,12 +7,11 @@ test_that('compactness works', {
     counties = rep(1, 25)
   )
   expected <- structure(list(district = c(1L, 2L, 3L, 1L, 2L, 3L), PolsbyPopper = c(
-    0.285408144266143,
-    0.246155768275462, 0.335907604022625, 0.397299799481666, 0.246155768275462,
-    0.234180177544589
+      0.28558422246899, 0.245854953175331, 0.336597469649693,
```

Comment:
So I think what we have to do here is skip_on_cran because this will fail until sf/s2 is updated, I think? Even though the numbers are very close

Reply:
Thank you. We've added that check to the setup. We're providing that many digits as the "true" value, but the call to `testthat::expect_equal` is only checking the first four digits, which is about the right point (I think) that we would be worried about, given the total scale of the measure.

Reply:
Makes sense, thanks!